src/openrct2/platform/Platform.Win32.cpp

```diff
@@ -156,25 +156,27 @@ namespace Platform
 
         SYSTEMTIME st;
         FileTimeToSystemTime(&ft, &st);
-        return &st;
+        return st;
     }
 
     std::string FormatShortDate(std::time_t timestamp)
     {
-        SYSTEMTIME * st = TimeToSystemTime(timestamp);
+        SYSTEMTIME st = TimeToSystemTime(timestamp);
 
         wchar_t date[20];
```

Comment:
Do we know that 20 bytes is enough for all locales?

Reply:
Well, a `wchar_t` is typically UTF-16 or even UTF-32, so it should be enough for basic representations. It might _not_ be enough if the short date format is set to print the full name of days, though — "Wednesday" is be 9 characters already. But I don't think that is very likely to happen…?